### PR TITLE
feat: back restore with multiple cluster support

### PR DESCRIPTION
**Pull request by JeffreyDallas**
## Description

This pull request changes the following:

* change restore process to three steps: restore clusters, restore network, restore configuration
* add flag optionsFile to provide customized command line argument during restore
* add example of multi-cluster backup and restore
* remove backup restore example
* moved some function from `default-one-shot.ts` to helper function so it can be reused by back & restore command

### Related Issues

* Closes #
* Related to #2784

### Pull request (PR) checklist

* \[ ] This PR added tests (unit, integration, and/or end-to-end)
* \[ ] This PR updated documentation
* \[ ] This PR added no TODOs or commented out code
* \[ ] This PR has no breaking changes
* \[ ] Any technical debt has been documented as a separate issue and linked to this PR
* \[ ] Any `package.json` changes have been explained to and approved by a repository manager
* \[ ] All related issues have been linked to this PR
* \[ ] All changes in this PR are included in the description
* \[ ] When this PR merges the commits will be squashed and the title will be used as the commit message, the 'commit message guidelines' below have been followed

### Testing

* \[ ] This PR added unit tests
* \[ ] This PR added integration/end-to-end tests
* \[ ] These changes required manual testing that is documented below
* \[ ] Anything not tested is documented

The following manual testing was done:

* TBD

The following was not tested:

* TBD

<details>
<summary>
Commit message guidelines
</summary>
We use 'Conventional Commits' to ensure that our commit messages are easy to read, follow a consistent format, and for automated release note generation. Please follow the guidelines below when writing your commit messages:

1. BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.  NOTE: currently breaking changes will only bump the MAJOR version.
2. The title is prefixed with one of the following:

| Prefix    | Description                                         | Semantic Version Update | Captured in Release Notes |
|-----------|-----------------------------------------------------|-------------------------|---------------------------|
| feat:     | a new feature                                       | MINOR                   | Yes                       |
| fix:      | a bug fix                                           | PATCH                   | Yes                       |
| perf:     | performance                                         | PATCH                   | Yes                       |
| refactor: | code change that isn't feature or fix               | none                    | No                        |
| test:     | adding missing tests                                | none                    | No                        |
| docs:     | changes to documentation                            | none                    | Yes                        |
| build:    | changes to build process                            | none                    | No                        |
| ci:       | changes to CI configuration                         | none                    | No                        |
| style:    | formatting, missing semi-colons, etc                | none                    | No                        |
| chore:    | updating grunt tasks etc; no production code change | none                    | No                        |

</details>

### CODE REVIEW

File: src/commands/flags.ts [hunk 1]
```diff
@@ -1736,6 +1736,29 @@ export class Flags {
     },
   };
 
+  public static readonly optionsFile: CommandFlag = {
```
Comment (jeromy-cannon) [inline src/commands/flags.ts]: missing update to:
```
  public static readonly allFlags: CommandFlag[] = [
```

File: src/commands/flags.ts [hunk 2]
```diff
@@ -1736,6 +1736,29 @@ export class Flags {
...
+    definition: {
+      describe:
+        'Path to YAML file containing component-specific deployment options (consensus, block, mirror, relay, explorer)',
+      defaultValue: '',
+      type: 'string',
+    },
+    prompt: undefined,
+  };
+
+  public static readonly metallbConfig: CommandFlag = {
```
Comment (jeromy-cannon) [inline src/commands/flags.ts]: missing update to:
```
  public static readonly allFlags: CommandFlag[] = [
```

File: src/commands/flags.ts (hunk 1 above)
Comment (JeffreyDallas) [reply src/commands/flags.ts]: Updated

File: examples/multicluster-backup-restore/README.md [hunk 3]
```diff
@@ -0,0 +1,524 @@
+# Multi-Cluster Backup and Restore Example
+
+This example demonstrates a complete multi-cluster backup and restore workflow for a Hiero network using Solo's `config ops` commands. It showcases an advanced deployment pattern with:
+
+1. **Dual-cluster deployment** - Consensus nodes distributed across two Kubernetes clusters
+2. **External PostgreSQL database** - Mirror node using external database for production-like setup
+3. **Complete component stack** - Consensus, block, mirror, relay, and explorer nodes
+4. **Full backup/restore cycle** - ConfigMaps, Secrets, Logs, State files, and database dumps
+5. **Disaster recovery** - Complete cluster recreation and restoration from backup
+
```
[the remark this answers is not in the document]
Comment (JeffreyDallas) [reply examples/multicluster-backup-restore/README.md]: updated

**Review by jeromy-cannon — DISMISSED** [review]
